src/actions/collapseContext.ts

```diff
@@ -69,13 +76,16 @@ const collapseContext = (state: State, { at }: Options) => {
       newValue: createId(), // unique value
       path: simplePath,
     }),
+    // Sort parent context if sort preference exists
+    sortId ? sort(parentId) : null,
```

Comment:
Only sort the parent if it's not already sorted (i.e. check the sort status of the parent with [getSortPreference](https://github.com/cybersemics/em/blob/a27f56c62a1a16d0b31435201cd77aed7ccd156e/src/selectors/getSortPreference.ts)).